packages/devtools/src/commands/elementClick.js

```diff
@@ -0,0 +1,39 @@
+import getElementTagName from './getElementTagName'
+import executeScript from './executeScript'
+import { ELEMENT_KEY } from '../constants'
+
+const SELECT_SCRIPT = 'return (function select (elem) { elem.selected = true }).apply(null, arguments)'
+
+export default async function elementClick ({ elementId }) {
+    const page = this.windows.get(this.currentWindowHandle)
+    const elementHandle = this.elementStore.get(elementId)
+
+    if (!elementHandle) {
+        throw new Error(`Couldn't find element with id ${elementId} in cache`)
+    }
+
+    /**
+     * in order to allow clicking on option elements (as WebDriver does)
+     * we need to check if the element is such a an element and select
+     * it instead of actually executing the click
+     */
+    const tagName = await getElementTagName.call(this, { elementId })
+    if (tagName === 'option') {
```

Comment:
I'm a bit worried about this change.
Isn't it possible that user may want to actually click on `option` element instead of making it selected?

Maybe select script should be used only when selectBy* commands are called?

Reply:
> Isn't it possible that user may want to actually click on `option` element instead of making it selected?

Well if the user clicks on the element he will select it which is what happens here.

Reply:
yeah, but some implementations expects mouse click event to be triggered.
We can do same for input type checkbox and just change `checked` flag instead of clicking on it...
I just wanted to say that it looks a bit hacky to me even though I fully agree that such implementation is way simpler. 
Click is click, and we are mixing click logic with selecting option.

Reply:
> Click is click, and we are mixing click logic with selecting option.

In my first implementation I had an actual click but unfortunately this was throwing an error which is why I switched to this option. I doubt that developer have click listeners on option elements. Usually you would register a onChange handler on the select element.

Reply:
I'm resolving this for now then if no one else minds